server-pro/docker-compose.test.yml

```
@@ -13,6 +13,8 @@ services:
       - PYTHON_VERSION_ALT
       - JUPYTER_PYTHON_VERSION
       - R_VERSION_ALT
+      - RSW_LICENSE
```

This RSW_LICENSE will not _actually_ work until we switch RSP to RSW. Should we switch back to RSP for now, and take the RSW bit as a part of follow-up? Alternatively, we might could do something like this to fill the gap:

```suggestion
      - RSP_LICENSE=${RSW_LICENSE}
```